pmm-app/selenoid-docker-compose.yaml

```diff
@@ -0,0 +1,35 @@
+version: '3'
+services:
+  codecept:
+    image: codeception/codeceptjs
+    container_name: codecept
+    environment:
+      - RUN_MULTIPLE=true
+      - CODECEPT_ARGS=parallel --steps --debug --grep '(?=.*)^(?!.*@visual-test)'
+    volumes:
+      - './:/tests/'
+    links:
+      - 'selenoid'
+    extra_hosts:
+      - pmm-server:${PMM_SRV_ADDR}
+    depends_on:
+      - selenoid
+    network_mode: bridge
+
+  selenoid:
```

Comment:
I think this is outdated, https://github.com/percona/pmm-qa/blob/master/docker-compose.yml selenoid-ui is linked with selenoid to observe test execution, I am not sure if that is needed?